apps/examples/testcase/ta_tc/mqtt/itc/itc_mqtt_main.c

```diff
@@ -225,15 +300,50 @@ void itc_mqtt_publish_p(void)
 	g_mqtt_client_handle = mqtt_init_client(&g_mqtt_client_config);
 	TC_ASSERT_NEQ("mqtt_init_client", g_mqtt_client_handle, NULL);
 	res = mqtt_connect(g_mqtt_client_handle, CONFIG_EXAMPLES_TESTCASE_ITC_MQTT_BROKER_ADDR, CONFIG_EXAMPLES_TESTCASE_ITC_MQTT_BROKER_PORT, 0);
-	TC_ASSERT_EQ("mqtt_connect", res, 0);
+	TC_ASSERT_EQ_CLEANUP("mqtt_connect", res, 0, mqtt_deinit_client(g_mqtt_client_handle));
 	ITC_MQTT_WAIT_SIGNAL;
 
 	res = mqtt_publish(g_mqtt_client_handle, ITC_MQTT_TOPIC, g_mqtt_msg, sizeof(g_mqtt_msg), 0, 0);
-	TC_ASSERT_EQ("mqtt_publish", res, 0);
+	TC_ASSERT_EQ_CLEANUP("mqtt_publish", res, 0,
+		mqtt_disconnect(g_mqtt_client_handle); mqtt_deinit_client(g_mqtt_client_handle));
 	ITC_MQTT_WAIT_SIGNAL;
 
 	res = mqtt_disconnect(g_mqtt_client_handle);
-	TC_ASSERT_EQ("mqtt_disconnect", res, 0);
+	TC_ASSERT_EQ_CLEANUP("mqtt_disconnect", res, 0, mqtt_deinit_client(g_mqtt_client_handle));
+	ITC_MQTT_WAIT_SIGNAL;
+
+	res = mqtt_deinit_client(g_mqtt_client_handle);
+	TC_ASSERT_EQ("mqtt_deinit_client", res, 0);
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @testcase             itc_mqtt_publish_n
+ * @brief                To check mqtt_connect
+ * @scenario             If used api returns not 0
+ * @apicovered           mqtt_publish
+ * @precondition         mqtt_init_client and mqtt_connect
+ * @postcondition        freeResources
+*/
+
+void itc_mqtt_publish_n(void)
+{
+	int res;
+	g_mqtt_client_handle = mqtt_init_client(&g_mqtt_client_config);
+	TC_ASSERT_NEQ("mqtt_init_client", g_mqtt_client_handle, NULL);
+	res = mqtt_connect(g_mqtt_client_handle,
+			CONFIG_EXAMPLES_TESTCASE_ITC_MQTT_BROKER_ADDR,
```

Comment:
Use short name for "CONFIG_EXAMPLES_TESTCASE_ITC_MQTT_BROKER_ADDR". Please make same for all testcase.